README.md

```diff
@@ -102,51 +104,36 @@ Example command:
 
 Clone this repository to your desired folder:
 
-<!--
-Example commands:
-
-```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
 ```
---->
+  git clone https://github.com/t-bello7/micro-blog.git
+```
 
 ### Install
 
 Install this project with:
 
-<!--
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/205749762-7e3ab0d3-0414-46c5-92d1-f4a9fcbe6054.png)

- [ ] Your link to your license file is not working :fearful: because you didn't add any license file :smile: It would be nice if you add a license file in your repository :+1: You can add a license file using these [guidlines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) :+1: